lib/builderator/tasks/packer.rb

```diff
@@ -28,6 +32,59 @@ def build(profile = :default, *args)
         invoke :configure, [profile], options
         run_with_input "#{Interface.packer.command} build - #{ args.join('') }", Interface.packer.render
       end
+
+      desc 'copy PROFILE *REGIONS', 'Copy AMIs generated by packer to REGIONS'
+      def copy(profile, *regions)
+        invoke :configure, [profile], options
+
+        regions.each do |region|
+          images.each do |image_name, image|
+            say_status :copy, "AMI #{image_name} (#{image.image_id}) from #{Config.aws.region} to #{region}"
+
+            client(region).copy_image(:source_region => Config.aws.region,
```

Comment:
I think we can use `ami_regions` in the packer.json to do this, given a recent version of packer.

https://www.packer.io/docs/builders/amazon-ebs.html#ami_regions